OpenSim/Simulation/Model/PrescribedForce.h

```diff
@@ -94,11 +89,19 @@ OpenSim_DECLARE_CONCRETE_OBJECT(PrescribedForce, Force);
     /** These are three functions providing the x,y,z measure numbers of the
     torque vector being applied to the body. The coordinate frame in which
     this vector is interpreted depends on the "torqueIsGlobal" property. **/
+
     // Would have been better for this to be a list property. 
     OpenSim_DECLARE_PROPERTY(torqueFunctions, FunctionSet,
         "Three functions describing the torque the PrescribedForce applies.");
+    /** The force applied by the PrescribedForce, this depends only on time */
+    OpenSim_DECLARE_OUTPUT(force_applied, SimTK::Vec3, getForceApplied, SimTK::Stage::Time);
```

Comment:
Would be good to know the reference frame in which the force is expressed in and indicating the effect of `forceIsGlobal`. Similarly for the torque and point.